lib/administrate/field/base.rb

```diff
@@ -16,6 +16,10 @@ def self.associative?
         self < Associative
       end
 
+      def self.eager_load?
+        self <= BelongsTo || self <= HasOne
+      end
```

Comment:
I think a better way to do this would be to have subclasses implement it. So the default here in `Field::Base` would be always `false`, but then `Field::BelongsTo#eager_load?` would always return `true` (same for `HasOne`). What do you think?

Reply:
Good OO 👍